packages/cli/bin/cli.js

```diff
@@ -123,7 +126,13 @@ export default async function main() {
       break;
     }
     case 'lab': {
-      await labCmd({ config, configPath, flags, logger });
+      try {
+        require.resolve('@terrazzo/token-lab');
```

Comment:
cc @lilnasy not a perfect solution, but manually requiring folks install this is a short-term solution to breaking the circular dep chain that prevents the CLI from being tested reliably, or having to pull from npm. Having to pull from published packages in the test suite could cause some regressions, and would limit us in what we’re able to test.

But once users install this I think the circular dep may still be there, so I’ll come up with another solution as this gets closer to a stable release (and we’ll celebrate and have lots of fanfare etc etc). For now, this just unblocks the CLI from releasing / auto-installing `@terrazzo/token-lab` for everyone

Reply:
This is great. Until stable release, whatever allows progress is a good solution.